tests/test_grant_tagger.py

```diff
@@ -0,0 +1,137 @@
+import pytest
+
+import pandas as pd
+
+from nutrition_labels.grant_tagger import GrantTagger
+
+
+training_data = pd.DataFrame(
```

Comment:
not that it makes a difference but you could use a fixture as well here. also a pattern I want to get into lately is that test data is defined in one place and all tests use them